win32ss/gdi/ntgdi/dibobj.c

```diff
@@ -275,7 +275,7 @@ IntSetDIBits(
     else
     {
         /* Compressed format without a size. This is invalid. */
-        DPRINT1("Compressed format without a size!");
+        DPRINT1("Compressed format without a size\n");
```

Comment:
```suggestion
        DPRINT1("Compressed format without size\n");
```

Reply:
Yeah, but not in this PR without author/maintainer approval.

Reply:
PROGRAMMER-field in header is empty. I do assume @DougLyons might have touched the file at some point.